### ISSUE TITLE
Consider more generally applicable superclasses Focus and Concentration 

### ISSUE THREAD

**Issue by stuartasutton** [CredentialEngine/Schema-Development#535]
Currently, CTDL has three properties to denote the primary and secondary focus of a [`ceterms:Credential`](http://purl.org/ctdl/terms/Credential): (1) [`ceterms:degreeMajor`](http://purl.org/ctdl/terms/degreeMajor), (2) [`ceterms:degreeMinor`](http://purl.org/ctdl/terms/degreeMinor), and (3) [`ceterms:degreeConcentration`](http://purl.org/ctdl/terms/degreeConcentration).  While these properties are tied to degrees, many professional certificates and certifications (e.g., medicine) and their domain languages embody various notions of primary focus and concentration/specialization of work/practice and study (see e.g., [MedBiquitous](https://www.medbiq.org/standards)). 

One solution would be to create super-properties that allow us to cluster (and further develop) these designations--something like:

>>Primary Focus
>>>Degree Major
>
>>Specialization
>>>Degree Minor
>>>Degree Concentration

In systems that don't care about clarifying these distinctions, the sub-property values can be rolled up ("dumb down" in Dublin Core language) to `Primary Focus` and `Specialization`.  With additional thought, there is likely wordsmithing to be done on property labels etc. 

Of course, consideration might be given to dropping the word "Degree" from the current three properties and generalizing their definitions; however, while this would be generalizing the properties as opposed to narrowing their intent and extension, it would likely require major surgery. 

These proposals aside, the problem needs attention.

**Comment by siuc-nate** [comment]
What would the distinction (in terms we could easily communicate to others) be between "primary focus" and "specialization"? I would think you could simplify it a bit by picking one or the other and making all 3 existing properties a subproperty of it (unless/until there is reason to have two).

**Comment by stuartasutton** [comment]
@siuc-nate, it needs to be two or all precision is lost if values are rolled up.

**Comment by siuc-nate** [comment]
Since concentrations are within majors, should Concentration be a subproperty of Primary Focus instead of Specialization? Perhaps then we could leave out Specialization altogether:
- Primary Focus
  - Degree Major
  - Concentration
- Degree Minor
Though I would still prefer putting all of them under one superproperty.

On a slightly related note, in reading up a bit more about majors, minors, and concentrations, I have to wonder if those are really aspects of the degree itself, or if they are actually some parallel thing (part of the _abstract_ learning opportunity? Part of the individual's chosen experiences (i.e. _instance_ of a learning opportunity)?) that are pursued simultaneously with the degree.  

Majors and Minors seem to be the same thing in the abstract (that is, the same list of choices), with the only differentiator being what the person pursuing the credential chose, rather than an aspect of the abstract degree.

I'll keep reading, but I wonder if we created those properties too hastily.

**Comment by siuc-nate** [comment]
The alternative to all of this would be the "major surgery" you mentioned - but that may be more correct in the end.  Say, for example, that we:
- Deprecate Degree Major
- Deprecate Degree Minor
- Reassign Concentration to be a property of Learning Opportunity instead of Credential
- Create some new property that reflects a list of available/offered majors and minors (since these lists seem to be the same(?)) for a Learning Opportunity ("Offered Areas Of Focus"? "Available Majors and Minors"?)
- Create some property that achieves the original goal at the credential level - assuming it is still correct to say that a singular **abstract** credential has a singular specialization and isn't just a separate, distinct credential from the "generic" version, if there is such a thing

That last one troubles me - if the person pursuing the credential chooses any of the properties being discussed (major, minor, concentration, specialization, focus, etc.) then those (or rather, any specific value/selection combination of those) aren't really a part of the _abstract_ credential that exists without the context of a pursuer - they're a part of a particular instance of a credential. At least that's the train of thought I'm having so far.

I'm willing to be talked down out of the tree, though.

**Comment by stuartasutton** [comment]
"That last one troubles me" too, because you made it up Nate. The following statement is not correct now or before: 

>"the original goal at the credential level - assuming it is still correct to say that a singular abstract credential has a singular specialization and isn't just a separate, distinct credential from the "generic" version, if there is such a thing."

What you are basically saying is that a degree with 5 concentrations, should be modeled as 5 different credentials. We've had glimmers of this conversation...taken to a logical conclusion, if the five flavors of the degree had 2 separate audiences (instate/out of state), we could effectively have 10 distinct degrees and, if pushed further in terms of other conditions, there'd be many more credentials and no need for the ConditionProfile because there'd be as many credentials as there were separate conditions. 

Can we put this conversation to bed...

**Comment by stuartasutton** [comment]
We did not create the current properties too hastily. We simply defined them to fit one context...so they are a bit narrow. That's not to say that doing that was wrong. It wasn't. We just have to provide the means for credentials in other context (professional credentialing) to also have specializations.

**Comment by siuc-nate** [comment]
>What you are basically saying is that a degree with 5 concentrations, should be modeled as 5 different credentials[...]

Only if the person who earned it wants to say that they earned a particular credential with a particular concentration via a single URI.  You are probably right about me getting my wires crossed in regards to the condition profile thing - that is what I was thinking of: the use case where someone wanted to indicate, via URI, the particular route they had taken to earn a credential.  I suppose that is a separate issue from this.

So in short, nevermind that part - but what about the other steps/options I outlined? Specifically:
- Do major/minor/concentration belong on credential or learning opportunity?
- Do we really need both "primary focus" and "specialization" (and what is the differentiating factor between them)?

**Comment by stuartasutton** [comment]
>"Only if the person who earned it wants to say that they earned a particular credential with a particular concentration via a single URI."

Nate, the anti keeps getting upped...who said anything about doing it with a 'single URI'? In a student's claim, what was earned could be identified as this credential (URI) with that concentration (URI). Since concentrations stem from a condition set (ConditionProfile), the following image from the Handbook (ctid's on ConditionProfile's added), would identify the resource descriptions under such a claim.  The person could assert that she earned a Bachelor of Arts in Communication (ce-d64419b0-314e-475f-b001-20124bd214c7) with a concentration in "Social and Cultural Communication" (ce-d64419b0-314e-475f-b001-20124bd214c7). 

![fig 2 2c-rev2](https://user-images.githubusercontent.com/2939046/41627238-55c78210-73d4-11e8-87cb-222f98129d79.png)

>"Do major/minor/concentration belong on credential or learning opportunity?"

They belong to the Credential. It's the Credential that orchestrates and contextualizes its major and any available minors or concentrations.

>"Do we really need both "primary focus" and "specialization" (and what is the differentiating factor between them)?"

Nate, I'm puzzled by the question. The point of property "dumb down" (roll up) is not to lose the distinctions entirely. E.g., If you dumbed everything down to "Focus" (or something like that), there would be no distinction at all.

**Comment by siuc-nate** [comment]
As I said, I had gotten my wires crossed with the other topic - my apologies.

As for the other properties, is the distinction that a specialization is something within a primary focus?  If so, then given that a Minor isn't necessarily "within" a Major, would it be semantically correct to put Minor under Specialization?

How about (since Major and Minor pull from the same list of options and are independent of one another as best I can tell):
- Degree Major
- Degree Minor
- Specialization
  - Degree Concentration

**Comment by stuartasutton** [comment]
Nate, we need to get back to the topic as proposed...many professional certificates and certifications (e.g., medicine) and their domain languages embody various notions of primary focus and concentration/specialization of work/practice and study. We do not accommodate them. The discussion regarding the MedBiquitous ecosystem mapping makes that quite clear. The proposal opened the possibility in contexts other than degrees (e.g., certifications) by creating super-properties that embody the general notions of each to which the current degree-related properties fit.

**Comment by siuc-nate** [comment]
Let's move forward with this with the August milestone.

**Comment by stuartasutton** [comment]
Yes, makes sense.

**Comment by siuc-nate** [comment]
Per our 7/17/2018 conversation:
Stuart will develop definitions for Primary Focus and Specialization that distinguish them from one another and can help explain them to others.

**Comment by siuc-nate** [comment]
This hasn't come up again since the middle of 2018. I recommend we drop it.
